lib/tasks/coronavirus.rake

```diff
@@ -23,4 +23,11 @@ namespace :coronavirus do
     puts "title: #{page.title}"
     puts "sections_title: #{page.sections_title}"
   end
+
+  desc "Sync timeline entries with entries in the yaml file"
+  task sync_timeline_entries: :environment do
+    CoronavirusPages::TimelineEntryBuilder.new.create_timeline_entries
+
+    puts "Timeline Entries synced for coronavirus landing page..."
+  end
```

Comment:
I'm not sure if it breaks convention, but it'd be good to have a test for the rake task - to be honest the amount of code here is sufficiently short it'd probably be reasonable for this to be written in the rake task, however I'm not sure if that breaks a convention.

Reply:
So I've basically just copied this from [here](https://github.com/alphagov/collections-publisher/blob/e47d09b50f80942e9db7eb30200c5fffe5524d0e/lib/tasks/coronavirus.rake#L31) hoping to follow convention :D. I think the rake task test should probably be added to another story if we feel strongly about it because I don't think I've added enough here to warrant it

Reply:
Sure, sounds like there isn't a convention for testing rake tasks. That's a shame but doesn't seem worth introducing one for this given it's only a temporary task.